UNRELEASED.md

```diff
@@ -36,6 +36,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Code quality
 
+- Use `import styles from './foo.scss` instead of non-standard `import * as styles from './foo.scss` when importing scss files ([#929](https://github.com/Shopify/polaris-react/pull/929))
```

Comment:
```suggestion
- Use `import styles from './foo.scss';` instead of non-standard `import * as styles from './foo.scss';` when importing scss files ([#929](https://github.com/Shopify/polaris-react/pull/929))
```